### PR TITLE
Issue/55/pipe fixes

### PR DESCRIPTION
**Pull request by eacharles**
## Code Quality
- [ ] My code follows [the code style of this project](https://rail-hub.readthedocs.io/en/latest/source/contributing.html#naming-conventions)
- [ ] I have written unit tests or justified all instances of `#pragma: no cover`; in the case of a bugfix, a new test that breaks as a result of the bug has been added
- [ ] My code contains relevant comments and necessary documentation for future maintainers; the change is reflected in applicable demos/tutorials (with output cleared!) and added/updated docstrings use the [NumPy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] Any breaking changes, described above, are accompanied by backwards compatibility and deprecation warnings

### CODE REVIEW

**Review by sschmidt23 — APPROVED** [review]
Looks good

File: src/rail/tools/photometry_tools.py [hunk 1]
```diff
@@ -420,16 +420,16 @@ class DustMapBase(RailStage):
 
     config_options = RailStage.config_options.copy()
     config_options.update(
-        bands=Param(str, default='ugrizy', msg="Names of the bands"),
```
[the remark this answers is not in the document]
Comment (eacharles) [reply src/rail/tools/photometry_tools.py:423]: It was redundant with the keys of the a_env dict...

[the remark this answers is not in the document]
Comment (eacharles) [reply src/rail/tools/photometry_tools.py:423]: So, better to only have one thing we have to update when we switch between instruments...

[the remark this answers is not in the document]
Comment (ztq1996) [reply src/rail/tools/photometry_tools.py:423]: Got it

**Pull request merged**